src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -43,6 +43,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
         AffiliationModifier.removeAffiliations(personToDelete.getAffiliations(), personToDelete, model);
+        AffiliationModifier.removeAffiliationHistory(personToDelete.getAffiliationHistory(), personToDelete, model);
```

Comment:
To clarify, the deletion of removed `Person` from `affiliationHistory` is all set?

Reply:
Yep, included in this PR, feel free to test it out and lmk if theres any issues!